README.md

```diff
@@ -2,34 +2,55 @@
 
 # qubes-template-securedrop-workstation
 
-This work was inspired by and reuses code from the Whonix Qubes template: https://github.com/adrelanos/qubes-template-whonix
-It is a derivative work under the GPL license, version 3 (see the files `COPYING` and `GPLv3` for details)
-
+Repository for managing the TemplateVM RPM used by the [SecureDrop Workstation] in provisioning custom VMs.
 
 ## Build instructions
 
-Note that these instructions must be carried out on a Fedora-based Qubes VM. Building templates uses a substantial amount of disk space.
+Note that these instructions must be carried out on a Fedora-based Qubes VM.
+Building templates uses a substantial amount of disk space.
 
 ### Set up build VM:
-1. Create a fedora-31 based AppVM for building the templates
-2. Increase the disk size to at least 20GB (as the build uses over 10GB): qvm-volume extend sd-template-builder:private 15GB (if your VM is not named sd-template-builder, adjust that command)
-3. Import the QubesOS master key and the GPG key used to sign tags (see https://www.qubes-os.org/security/verifying-signatures/)
+Set up a long-lived VM that you can use for building SDW templates.
+You'll only need to perform this step once, although make sure to check
+whether your Fedora version remains current.
+
+1. Create an AppVM based on the most recent fedora release: `qvm-create --label purple --template fedora-33 sd-template-builder`
+2. Increase the disk size to at least 20GB (as the build uses over 10GB): `qvm-volume resize sd-template-builder:private 20G`
+3. Clone this repository into the AppVM: `git clone https://github.com/freedomofpress/qubes-template-securedrop-workstation`
 
 ### Automatic build
+We maintain a wrapper script that handles the interoperation with the upstream [qubes-builder] logic.
+Typically, you'll need only this short-and-sweet workflow to build a new template RPM.
+If you encounter problems, see the manual build instructions below.
 
 1. `make template`
-2. The Template RPM can be found in ./qubes-builder/qubes-src/linux-template-builder/rpm/
+2. The Template RPM can be found in `./qubes-builder/qubes-src/linux-template-builder/rpm/`
 
 ### Testing changes to builder logic
 
-1. Repace the following value in `securedrop-workstation.conf`: ` BRANCH_template_securedrop_workstation ?= $TEST_BRANCH`
-2. Sign the tag using a trusted key (or edit the `builder/build-workstation-template` script to import/trust the key)
-3. `make template`
+The qubes-builder logic expects signed tags on the most recent HEAD commit of the target branch.
```

Comment:
it wouldn't hurt to mention one more time that the qubes-builder expects this singed tag to be pushed to remote (it's mentioned below but it's an important enough detail to mention here too i think)